src/transformers/models/auto/tokenization_auto.py

```diff
@@ -166,6 +166,7 @@
                     "MBart50TokenizerFast" if is_tokenizers_available() else None,
                 ),
             ),
+            ("mctct", ("Wav2Vec2CTCTokenizer", None)),
```

Comment:
This enables the pipeline to automatically load the tokenizer for you.

Reply:
There should be a default tokenizer for the model type anyway, as users might not always have the class properly set in the config/tokenizer config.

So this change should stay regardless of whether we keep the pipeline or not!